cmd/snap/cmd_auto_import.go

```diff
@@ -366,7 +374,20 @@ func (x *cmdAutoImport) Execute(args []string) error {
 		return err
 	}
 
-	if added1+added2 > 0 {
+	// nothing imported, no further action needed
+	if added1+added2 == 0 {
+		return nil
+	}
+
+	// only try to create users for unmanaged devices
+	isManaged, err := x.isManaged()
```

Comment:
My preference would be to make this a problem of the daemon, by having a new flag option "automatic" in the request. It would also replace for this use case the other current flags we set, that would make it easier to support later as we discussed recently having system-user users that are not sudoers. With the current approach the decision is done here which would be the wrong place then.